tests/hw-accel/kmm/internal/get/get.go

```
@@ -397,6 +397,27 @@ func DRADaemonSet(apiClient *clients.Settings, moduleName,
 
+// AllDRADaemonSets returns all DRA DaemonSets for a given module in a namespace.
+func AllDRADaemonSets(apiClient *clients.Settings, moduleName,
+	nsName string) ([]appsv1.DaemonSet, error) {
+	dsList, err := apiClient.K8sClient.AppsV1().DaemonSets(nsName).List(
+		context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		return nil, fmt.Errorf("error listing DaemonSets: %w", err)
+	}
+
+	prefix := moduleName + "-dra-"
+	var result []appsv1.DaemonSet
```

golangci-lint (2.12.2)

[error] missing whitespace above this line (never cuddle decl)

(wsl_v5)